csuntechlab/iSTART
# STR-48 -- FE: Add notification to ensure cookies are enabled

# Description
Should notify users to accept cookies policy; should not log user in if policy not accepted.
  
# Testing
#### Testing Notification Display
1. Visit iSTART and clear application storage ***`localStorage & cookies`*** .
2. Reload the page and ensure the `cookies notification` appears at the bottom.
3. Test the responsiveness of the `cookies notification`.

#### Testing Notification Logic
1. Attempt to login and notice you aren't able to if you haven't accepted the policy.
2. Click ***accept*** on the `cookies notification` and re-attempt to login.
3. Ensure you are able to login once the cookies policy is ***accepted*** .
4. Logout
5. Clear the application storage again and ***disable cookies*** .
6. Ensure the `cookies notification` appears but this time notice the ***accept*** button is now asking to ***enable cookies***.
7. Attempt to login and ensure you aren't able to login.
8. Try to break the notification logic by doing weird stuff.
